### PR TITLE
Initialize the search box with last searched string

### PR DESCRIPTION
**Pull request by MeetM**
The string is selected by default. Similar to what IntelliJ does.
screen shot 2018-03-03 at 1 35 51 pm

### CODE REVIEW

File: quick-tabs/popup.js [hunk 1]
```diff
@@ -357,11 +357,28 @@ $(document).ready(function() {
...
+  /**
+   * Try to fetch the last search string.
+   * If present, use it to render only matched tabs list
+   * else, render all current tabs list
+   */
+  chrome.storage.sync.get(["qtLastSearchString"], function(items) {
+    var str = items["qtLastSearchString"];
+    if (typeof str !== "undefined" && str.length > 0) {
+      $("#searchbox").val(str);
+      $("#searchbox").select();
```
Comment (matharden) [inline quick-tabs/popup.js]: Rather than searching the DOM again, you could chain `.select()` onto the previous line.

**Comment by matharden** [comment]
👍  Should be optional though.

File: quick-tabs/popup.js (hunk 1 above)
Comment (MeetM) [reply quick-tabs/popup.js]: Thanks! Will make the change.

**Comment by MeetM** [comment]
> 👍 Should be optional though.

IntelliJ has it ON by default, and it seems very natural after a while.
Since the _last searched string_ is pre-selected, it doesn't even require any additional key-press before typing a new one.

Not against having it as an option, but I think it should be ON by default

**Comment by MartinLichtblau** [comment]
And why isn't it merged?

**Comment by MeetM** [comment]
Updated pull request - Added an option to disable last-searched string from search box

PS - Looks like this repo and extension are no longer maintained :(

File: quick-tabs/popup.js (hunk 1 above)
Comment (MeetM) [reply quick-tabs/popup.js]: Made the changes

**Pull request merged**